uber/templates/fields/attendee.html

```diff
@@ -868,7 +870,7 @@ <h4>{{ tier.name }}</h4>
   {% call macros.read_only_if(read_only, attendee.birthdate, post_text=post_text) %}
     <div class="col-sm-6">
       <input type='text' class="form-control date" name="birthdate" value="{{ attendee.birthdate|datetime("%Y-%m-%d") }}"/>
-    <span class="form-control-static">{{ post_text }}</span>
+    {% if post_text %}<span class="form-control-static">{{ post_text }}</span>{% endif %}
```

Comment:
Thanks to the base.html-level javascript that hides blank read-only fields, this field was getting hidden if there was no post_text!